planet/js/helper.js

```diff
@@ -146,8 +146,8 @@ $(document).ready(() => {
     });
     // eslint-disable-next-line no-unused-vars
     document.getElementById("view-more-chips").addEventListener("click", (evt) => {
-        const showMore = _("Show more tags");
-        const showLess = _("Show fewer tags");
+        const showMore = _("Show more tags ▼");
```

Comment:
You put the ▼ outside the string in StringHelper, but you need to do it here as well.